docker/unlock-app.dockerfile

```diff
@@ -16,7 +16,7 @@ WORKDIR /home/unlock/unlock-app
 COPY --chown=node unlock-app/ /home/unlock/unlock-app/.
 RUN npm run build
 
-# Copy the .git stuff which is required by chromatic.
+# Copy the .git stuff
```

Comment:
Do we still actually need these parts if they were originally from chromatic?

Reply:
I was wondering but I think it's actually good practice to have them in there so we can easily see what commits is being run when debugging things in ci.